Ginger/GingerCoreCommon/Repository/RepositoryItemBase.cs

```diff
@@ -1072,6 +1074,10 @@ private void DirtyCheck(string name)
 
         internal void RaiseDirtyChanged(object sender, EventArgs e)
         {
+            if (!SourceDirtyQueue.Contains(sender as RepositoryItemBase))
+            {
+                SourceDirtyQueue.Enqueue(sender as RepositoryItemBase);
```

Comment:
I don't see SourceDirtyQueue is being used anywhere, do we need it?

Reply:
is part of displaying what was changed in the repository item, removed it as of now as there is some more work to do on that area